generator/Dockerfile

```diff
@@ -1,13 +1,15 @@
 # (c) Copyright 2019-2023 OLX
-FROM rust:1.66.0-alpine3.17
+FROM rust:1.73.0-alpine3.18
 
-ENV VIPS_VERSION=8.13.3
+ENV VIPS_VERSION=8.14.5
 
 RUN apk add --update --no-cache --repository=https://dl-cdn.alpinelinux.org/alpine/v3.17/main \
```

Comment:
@negezor - thanks for the work you have done with the PR.

> RUN apk add --update --no-cache --repository=https://dl-cdn.alpinelinux.org/alpine/v3.17/main \

Can you bump this to *3.18* to match the `FROM`